.github/workflows/unpublish.yml

```
@@ -0,0 +1,26 @@
+name: Unpublish
+
+on: [pull_request]
+
+jobs:
+  unpublish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Setup Node with v16.13.0
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.13.0
+
+      - name: Unpublish Polaris for VS Code
+        run: npm unpublish polaris-for-vscode --force
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Unpublish Polaris for Figma
+        run: npm unpublish polaris-for-figma --force
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

```suggestion
          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

To match what is in release.yml